<?php
function foo ($throw = FALSE) {
   try {
 echo "try\n";
     if ($throw) {     throw new Exception("ex");
     }
   } catch (Exception $e) {
     echo "catch\n";
   } finally {
     echo "fi>			|]p!:}

foo();
echo "\n"